include/lsst/jointcal/Associations.h

```diff
@@ -121,6 +121,14 @@ class Associations {
      */
     void selectFittedStars(int minMeasurements);
 
+    /**
+     * Make fitted star positions and fluxes be the average of their measured stars.
+     *
+     * Only call after selectFittedStars() has been called: it assumes that each measuredStar points to a
+     * fittedStar, and that the measurementCount for each fittedStar is correct.
```

Comment:
This kind of requirement would be a good argument for making the everything part of `Associations` initialization. But maybe on another ticket. 🙂

Reply:
Unfortunately, this can't reasonably be done as part of init. Instead, I've made a `prepareFittedStars` that does both of these steps, and made the two methods it calls private.